src/Pages/Home/EventPage/EventPage.module.scss

```diff
@@ -0,0 +1,167 @@
+@import url('https://fonts.googleapis.com/css2?family=JetBrains+Mono&display=swap');
+@font-face{
+    font-family: "Pixeloid";
+    src: local("PixeloidSans"),
+    url("../../../../public/Font/PixeloidSans.ttf") format("truetype");
+}
+@font-face{
+    font-family: "PixeloidBold";
+    src: local("PixeloidSans-Bold"),
+    url("../../../../public/Font/PixeloidSans-Bold.ttf") format("truetype");
```

Comment:
change to `/fonts/[path]`. files in public directory are served at the root path.